src/pages/contract/_components/ContractTabs.tsx

```diff
@@ -0,0 +1,115 @@
+import clsx from "clsx";
+import { Dispatch, SetStateAction } from "react";
+
+interface ContractOptionsProps {
+  selectedTab: ContractOptionsTitle;
+  setSelectedTab: Dispatch<SetStateAction<ContractOptionsTitle>>;
+}
+
+export default function ContractTabs({
+  selectedTab,
+  setSelectedTab,
+}: ContractOptionsProps) {
+  return (
+    <div className="flex flex-row gap-x-4 md:pt-[3.67px] pt-[19.67px]">
+      {selectedTab === ContractOptionsTitle.Code ? (
+        <ButtonTab
+          testId="contract-code-options-clicked-title"
+          active
+          tab={ContractOptionsTitle.Code}
+          setSelectedTab={setSelectedTab}
+        />
+      ) : (
+        <ButtonTab
+          testId="contract-code-options-title"
+          active={false}
+          tab={ContractOptionsTitle.Code}
+          setSelectedTab={setSelectedTab}
+        />
+      )}
+      {selectedTab === ContractOptionsTitle.Transactions ? (
+        <ButtonTab
+          testId="contract-transaction-options-clicked-title"
+          active
+          tab={ContractOptionsTitle.Transactions}
+          setSelectedTab={setSelectedTab}
+        />
+      ) : (
+        <ButtonTab
+          testId="contract-transaction-options-title"
+          active={false}
+          tab={ContractOptionsTitle.Transactions}
+          setSelectedTab={setSelectedTab}
+        />
+      )}
+      {selectedTab === ContractOptionsTitle.Tokens ? (
+        <ButtonTab
+          testId="contract-tokens-options-clicked-title"
+          active
+          tab={ContractOptionsTitle.Tokens}
+          setSelectedTab={setSelectedTab}
+        />
+      ) : (
+        <ButtonTab
+          testId="contract-tokens-options-title"
```

Comment:
can we combine these button? we can add the ternary operator in the arguments
e.g.: testId = `${selectedTab === x} ? testIdA : testIdB`